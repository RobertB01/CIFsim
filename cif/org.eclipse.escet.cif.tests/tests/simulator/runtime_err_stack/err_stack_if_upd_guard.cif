//////////////////////////////////////////////////////////////////////////////
// Copyright (c) 2010, 2022 Contributors to the Eclipse Foundation
//
// See the NOTICE file(s) distributed with this work for additional
// information regarding copyright ownership.
//
// This program and the accompanying materials are made available
// under the terms of the MIT License which is available at
// https://opensource.org/licenses/MIT
//
// SPDX-License-Identifier: MIT
//////////////////////////////////////////////////////////////////////////////

automaton p:
  disc real x;
  location:
    initial;
    edge do if 1 / 0 > 0: x := 0.0
            end;
end
