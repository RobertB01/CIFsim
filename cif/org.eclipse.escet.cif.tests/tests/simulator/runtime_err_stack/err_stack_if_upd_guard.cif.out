Initial state: time=0.0, p=*, p.x=0.0

